app/Repository/TalkRepository.php

```diff
@@ -56,4 +56,47 @@ public function getTotalTalks(\DatePeriod $datePeriod): int
 
         return (int) $queryBuilder->getQuery()->getSingleScalarResult();
     }
+
+    public function findTalkStatisticsPerDay(\DatePeriod $datePeriod): array
+    {
+        return $this->findTalkStatistics($datePeriod, [
+            'year' => 'YEAR(o.startsAt) AS year',
+            'month' => 'MONTH(o.startsAt) AS month',
+            'day' => 'DAY(o.startsAt) AS day',
+        ]);
+    }
+
+    public function findTalkStatisticsPerMonth(\DatePeriod $datePeriod): array
+    {
+        return $this->findTalkStatistics($datePeriod, [
+            'year' => 'YEAR(o.startsAt) AS year',
+            'month' => 'MONTH(o.startsAt) AS month',
+        ]);
+    }
+
+    private function findTalkStatistics(\DatePeriod $datePeriod, array $groupBy): array
+    {
+        $queryBuilder = $this->createQueryBuilder('o');
+
+        $queryBuilder
+            ->select('COUNT(o.id) AS total')
+            ->andWhere(
+                $queryBuilder->expr()->gte('o.startsAt', ':start_date'),
+            )
+            ->andWhere(
+                $queryBuilder->expr()->lt('o.startsAt', ':end_date'),
+            )
+            ->setParameter('start_date', $datePeriod->getStartDate())
+            ->setParameter('end_date', $datePeriod->getEndDate())
```

Comment:
Discussed snake_case vs camelCase for doctrine parameters in one of the previous PRs.

```suggestion
                $queryBuilder->expr()->gte('o.startsAt', ':startDate'),
            )
            ->andWhere(
                $queryBuilder->expr()->lt('o.startsAt', ':endDate'),
            )
            ->setParameter('startDate', $datePeriod->getStartDate())
            ->setParameter('endDate', $datePeriod->getEndDate())
```